.github/workflows/test.yml

```diff
@@ -40,14 +40,29 @@ jobs:
           sudo apt update
           sudo apt install libgdcm-dev
 
+      # install from source, because apt has old version
+      - name: Install GoogleTest and GoogleMock
+        run: |
+          git clone https://github.com/google/googletest.git -b v1.13.0
+          cd googletest
+          mkdir build && cd build
+          cmake ..
+          make
+          sudo make install
```

Comment:
We also need an instruction for developers to get and build the googletest framework locally without doing sudo make install.

Reply:
I can add it, though I believe it would be more suitable for the installation section of documentation. Not in the readme, because I think this is too detailed information for a readme file and it would grow too big. I can add it in another PR with documentation :)

Reply:
See https://github.com/DataMedSci/yagit/issues/58